librz/util/buf.c

```diff
@@ -270,7 +270,14 @@ static RzBuffer *new_buffer(RzBufferType type, void *user) {
 	return rz_buf_new_with_methods(methods, user);
 }
 
-RZ_API RzBuffer *rz_buf_new_empty(ut64 len) {
+/**
+ * \brief Creates a new empty buffer with a predefined size;
+ * \param len The length in byte of the new buffer.
+ * \return Return the new allocated buffer.
+ *
+ * The function creates a new allocated buffer using the RZ_BUFFER_BYTES back end.
```

Comment:
The function creates a new buffer of the specified length in memory, filled with `\0`.